app/models/miq_server.rb

```diff
@@ -491,15 +491,15 @@ def self.managed_resources
     {
       :vms                     => Vm.active.count,
       :hosts                   => Host.active.count,
-      :aggregate_physical_cpus => MiqRegion.my_region.aggregate_physical_cpus(Host.active),
+      :aggregate_physical_cpus => Host.active.in_my_region.sum(:aggregate_physical_cpus),
```

Comment:
these are great

Reply:
or they would be if they weren't failing, i did really want `.sum(&:...)`

Reply:
I hope you don't mean `.sum(&:..)`, and by that I mean with a proc.  Should just be passing the field name with that, pretty sure.

Reply:
yeah, we call `@klass.arel_attribute(:aggregate_physical_cpus)` which is nil right before `operation_over_aggregate_column` and then `undefined method sum for nil`, fun.